types/material-ui/index.d.ts

```diff
@@ -5862,8 +5860,8 @@ declare module 'material-ui/svg-icons/av/library-books' {
 }
 
 declare module 'material-ui/svg-icons/av/art-track' {
-    export import AvArtTrack = __MaterialUI.SvgIcon;
-    export default AvArtTrack;
+    export import AlettTrack = __MaterialUI.SvgIcon;
+    export default AlettTrack;
```

Comment:
Haha nice catch! I fixed it with https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18189/commits/c46fa7bb9aee52301e14fe2d7ca4861f2a5d8efa